### PR TITLE
Fixes to Ukrainian data

### PR DESCRIPTION
**Pull request by isamokhin**
Dateparser does not recognize the form "року" ("of year") and a few more forms commonly used in Ukrainian dates. I added several of them to uk.yaml and generated new uk.py with the provided script, and updated the tests accordingly.

### CODE REVIEW

**Review by Gallaecio — APPROVED** [review]
Looks great to me, as someone who does _not_ speak Ukranian.

File: tests/test_languages.py [hunk 1]
```diff
@@ -1605,6 +1611,10 @@ def test_translation(self, shortname, datetime_string, expected_translation):
         param('uk', "18 хвилин тому", "18 minute ago"),
         param('uk', "через 22 роки", "in 22 year"),
         param('uk', "цього тижня", "0 week ago"),
+        param('uk', "півгодини тому", "30 minute ago"),
+        param('uk', "пів години тому", "30 minute ago"),
+        param('uk', "півроку тому", "6 month ago"),
```
Comment (Alexandr1988) [inline tests/test_languages.py:1616]: 6 month**S** ago

File: tests/test_languages.py [hunk 2]
```diff
@@ -1605,6 +1611,10 @@ def test_translation(self, shortname, datetime_string, expected_translation):
         param('uk', "18 хвилин тому", "18 minute ago"),
         param('uk', "через 22 роки", "in 22 year"),
         param('uk', "цього тижня", "0 week ago"),
+        param('uk', "півгодини тому", "30 minute ago"),
+        param('uk', "пів години тому", "30 minute ago"),
```
Comment (Alexandr1988) [inline tests/test_languages.py:1615]: 30 minute**S** ago

File: tests/test_languages.py [hunk 3]
```diff
@@ -1605,6 +1611,10 @@ def test_translation(self, shortname, datetime_string, expected_translation):
         param('uk', "18 хвилин тому", "18 minute ago"),
         param('uk', "через 22 роки", "in 22 year"),
         param('uk', "цього тижня", "0 week ago"),
+        param('uk', "півгодини тому", "30 minute ago"),
```
Comment (Alexandr1988) [inline tests/test_languages.py:1614]: 30 minute**S** ago

File: tests/test_languages.py (hunk 1 above)
Comment (isamokhin) [reply tests/test_languages.py:1616]: Please look at other test cases for all other languages. The goal of the library is not to translate dates into grammatically correct English sentences, it's just a convenient step before converting them into datetime objects.

**Comment by noviluni** [comment]
Hi @isamokhin!
I checked the content with a native and it seems that all is correct :)

However, I will wait until we fix the pipelines: https://github.com/scrapinghub/dateparser/pull/859 before merging this.

Thank you! :muscle:

**Review by noviluni — APPROVED** [review]

**Comment by noviluni** [comment]
(I closed and opened the PR to trigger the pipelines)

**Comment by noviluni** [comment]
It seems that the tests are failing: `tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_relative_future_dates_212__3`

**Comment by isamokhin** [comment]
Hi!

I checked everything, and apparently the problem is that with addition of "за" for Ukrainian (meaning "in" in many Slavic languages), test for `param("за 3 годину", in_future={'years': 3}, period='year')` interpreted it as Ukrainian instead of Serbian, and translated годину as hour (Ukrainian meaning), instead of year (Serbian meaning).
I can remove "за" from Ukrainian data for the moment, but in general "за" is used in texts and should be kept. What can we do when translation is wrong because of different translations of the same word, like here?

**Comment by Gallaecio** [comment]
@isamokhin Thank you!

@serhii73 completed your work in #1027, moving the failing Serbian test to another test where input language is explicitly passed to Dateparser.

**Pull request closed without merging**